trinity/config.py

```diff
@@ -216,18 +218,19 @@ class TrinityConfig:
 
     def __init__(self,
                  network_id: int,
+                 app_identifier: str="",
                  genesis_config: Dict[str, Any]=None,
                  max_peers: int=25,
                  trinity_root_dir: str=None,
                  data_dir: str=None,
                  nodekey_path: str=None,
-                 logfile_path: str=None,
```

Comment:
Maybe at one point we allowed setting the `logfile_path` vial CLI but it doesn't seem like that functionality still exists. It seems to be dead code that was never called from anywhere.

Reply:
Then we should definitely remove the code that crashes if the log folder is missing